source/dashboards/cards.markdown

```diff
@@ -112,7 +112,7 @@ In [sections view](/dashboards/sections/), you can resize cards. Follow these st
 
 If you previously [resized](#resizing-a-card) a card in the [sections view](/dashboards/sections/), and you don't like the new size, you can revert back to the card's default size. Follow these steps:
 
-1. On the **Layout** tab, select the <iconify-icon inline icon="mdi-restore"></iconify-icon> icon.
+1. On the **Layout** tab, select the {% icon "mdi-restore" %} icon.
```

Comment:
```suggestion
1. On the **Layout** tab, select the {% icon "mdi:restore" %} icon.
```
this should be `mdi:restore` right?
Can we enhance the `SYNTAX` to blow up on wrong usage?

Reply:
This is wrong usage indeed, it could be right usage